.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2.1
+
+workflows:
+  ci:
+    jobs:
+      - test
```

Comment:
N.B.: We might need a context declared here to get the ambient OIDC token: https://circleci.com/docs/2.0/openid-connect-tokens/#openid-connect-id-token-availability